x-pack/plugins/lens/public/xy_visualization/to_expression.ts

```diff
@@ -118,8 +119,22 @@ export const buildExpression = (
             },
           ],
           fittingFunction: [state.fittingFunction || 'None'],
-          showXAxisTitle: [state.showXAxisTitle ?? true],
```

Comment:
I decided here instead of having 3 booleans to check the visibility of each axis to change it to axisTitlesVisibilitySettings in order to be in line with tickLabelsVisibilitySettings and gridlinesVisibilitySettings as all of them have to do with visibility settings. If there is any objection, I am ok to revert it to the previous one.